src/main/java/com/github/admissionCommittee/service/UserService.java

```diff
@@ -22,24 +23,30 @@
         .STUDENT;
 
 public class UserService extends GenericService<User> {
-
-    public SheetService sheetService = new SheetService();
+    private UserValidatorUtil validator = new UserValidatorUtil();
+    private SheetService sheetService = new SheetService();
 
     public UserService() {
         super(User.class, DaoFactory.getDaoFactory().getUserDao());
     }
 
+    @Override
+    public void save(User instance) {
+        validator.validate(instance);
+        super.save(instance);
+    }
+
+    @Override
+    public void save(List<User> instance) {
+        instance.forEach(validator::validate);
+        super.save(instance);
```

Comment:
if at list one user is not valid this method will not save any

Reply:
This method is for initialization and testing only, therefore, if validation fails, we just need to fix the validation algorithm or input data (if fault is in input). 
For production use, we can easily overload the method with filter.

Reply:
there is no documentation about that, so for me and other programmers its public method of CRUDrepository which we always use in production not only for initialization, also there is no throws segment and if thes method will thow sth that kill my site I will be very surprised

Reply:
Documentation added.